src/components/SoqlListView.component

```diff
@@ -29,148 +29,137 @@
+                    </apex:repeat>         
+                </tr>
+            </thead>
+            <tbody>
+                <apex:repeat value="{!listSLVObj}" var="item" rows="{!cRecordsPerPage}" >
+                    <tr class="slds-cell-shrink">
+                        <td>
+                            <apex:inputCheckbox value="{!item.checked}" styleClass="cbxSLVObj" />
+                        </td>
+                        <td class="slds-text-body--small">
+                            <apex:outputLink value="/{!item.sobj['id']}" >
+                                <apex:outputField value="{!item.sobj[pageController.strNameField]}" />
+                            </apex:outputLink>
+                        </td>
+                        <apex:repeat value="{!pageController.strFields}" var="f2" > 
+                            <apex:outputText rendered="{!f2 != pageController.strNameField}" >
+                                <td class="slds-text-body--small slds-cell-wrap">
+                                    <apex:outputField value="{!item.sobj[f2]}" />
+                                </td>
+                            </apex:outputText>
+                        </apex:repeat>
+                    </tr>
+                </apex:repeat>
+           </tbody>
+        </table>
+        <div class="PagerUI">
+            <apex:panelGrid columns="12" style="width:40%;"> 
+                <apex:outputText value="" />
+                <apex:commandLink action="{!setCon.first}" rerender="panelData" status="statusLoad" >{!$Label.labelListViewFirst}</apex:commandlink>
+                <apex:outputText value=" | " />
+                <apex:commandLink action="{!setCon.previous}" rerender="panelData" status="statusLoad" rendered="{!setCon.HasPrevious}">{!$Label.labelListViewPrevious}</apex:commandlink> 
+                <apex:outputText rendered="{!setCon.HasPrevious}" value=" | " />
+                <apex:commandLink action="{!setCon.next}" rerender="panelData" status="statusLoad" rendered="{!setCon.HasNext}">{!$Label.labelListViewNext}</apex:commandlink>
+                <apex:outputText rendered="{!setCon.HasNext}" value=" | " />
+                <apex:commandLink action="{!setCon.last}" rerender="panelData" status="statusLoad" >{!$Label.labelListViewLast}</apex:commandlink>
+                <apex:outputText value="" />
+                <apex:outputText value="{!$Label.labelListViewPageInfo}" escape="false" >
+                    <apex:param value="{!setCon.PageNumber}" />
+                    <apex:param value="{!NumberOfPages}" />
+                    <apex:param value="{!NumberOfItems}" />
+                </apex:outputText> 
+                <apex:outputText value="" />
+                <apex:commandLink action="{!showMoreRecordsPerPage}" rerender="panelData" status="statusLoad" >{!$Label.labelShowMore}</apex:commandlink>                   
+            </apex:panelGrid> 
+        </div>
+    </apex:outputPanel>
```

Comment:
looking better, but the spacing of page n of m is still a little weird.
![image](https://cloud.githubusercontent.com/assets/908119/21323531/c37f3e80-c5d2-11e6-8b88-6020edbb5508.png)
ideally it would be 

First | Prev | Next | Last   Page N of M   Show more

with just a single nbsp between each of the first/prevent/next/last and then a couple of nbsp's to move Page N of M over, and then a couple of npbp's Show more.